Emitron/Emitron/UI/Library/Filtering/FiltersView.swift

```diff
@@ -106,8 +106,7 @@ struct FiltersView: View {
   }
   
   private func applyOrCloseButton() -> MainButtonView {
```

Comment:
Sine this button no longer changes the title between apply or close, it should probably be renamed `applyFiltersButton()`

Reply:
Hey @0xTim  thanks for the comments will make another PR for changing the name of function.